<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZoomSettingDialog</class>
 <widget class="QDialog" name="ZoomSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>376</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">#dialogContainer {
	background-color: white;
	border-radius: 12px;
	border: none;
}

#bodyContainer {
	border-bottom-left-radius: 12px;
	border-bottom-right-radius: 12px;
}


QLabel {
	color: #1A1A1A;
	font-family: Open Sans;
	font-weight: 600;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
}

QLineEdit {
	color: #1A1A1A;
	background: white;
	font-family: Open Sans;
	font-weight: 400;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
	border: 1px solid #CCCCCC;
	padding-left: 12px;
	padding-right: 12px;
}

QTextEdit {
	color: #1A1A1A;
	background: white;
	font-family: Open Sans;
	font-weight: 400;
	font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
	border: 1px solid #CCCCCC;
	padding-left: 12px;
	padding-right: 12px;
}

QComboBox {
    background-color: white;
    border: 1px solid #ccc;
    border-radius: 8px;
    padding: 4px;
    font-size: 16px;
	line-height: 24px;
	letter-spacing: 0%;
    color: #333;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border: none;
}

QComboBox::down-arrow {
    image: url(:/images/chevron-down.svg);
    width: 16px;
    height: 16px;
}

QComboBox::down-arrow:on {
    image: url(:/images/chevron-up.svg);
}

QComboBox QAbstractItemView {
    background-color: white;
    margin-top: 0px;
    border: 1px solid #ccc;
    border-radius: 8px;
	outline: none;
}

/* Each dropdown item */
QComboBox QAbstractItemView::item {
	color: #1A1A1A;
    padding: 6px 12px;
    border-radius: 4px;
}

/* Hovered item */
QComboBox QAbstractItemView::item:hover {
    background-color: #e6f2ff;
}

/* Selected item */
QComboBox QAbstractItemView::item:selected {
    background-color: #d0e8ff;
    color: #1a1a1a;
}

#cancelButton {
	color: #1A1A1A;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 18px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
	border: 2px solid #1A1A1A
}

#cancelButton:hover {
    background-color: #D6ECFF;
}

#cancelButton:pressed {
    background-color: #B7DCFF;
    padding-top: 2px;
	padding-left: 2px;
}

#applyButton {
    background-color: #FDB515;
	color: white;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 18px;
	line-height: 24px;
	letter-spacing: 0%;
	border-radius: 8px;
}

#applyButton:hover {
    background-color: #E6A512;
}

#applyButton:pressed {
    background-color: #CC8F0F;
    padding-top: 2px;
	padding-left: 2px;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QWidget" name="dialogContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>684</width>
       <height>314</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>684</width>
       <height>314</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>18</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>64</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>18</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>16</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>16</number>
         </property>
         <item>
          <widget class="QLabel" name="dialogTitle">
           <property name="styleSheet">
            <string notr="true">color: black;
font-family: Open Sans;
font-weight: 500;
font-size: 22px;
line-height: 30px;
letter-spacing: 0%;
</string>
           </property>
           <property name="text">
            <string>Zoom In/Out</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="closeButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../Resources/Resources.qrc">
             <normaloff>:/images/Project/cross-icon.svg</normaloff>:/images/Project/cross-icon.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>18</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="scaleFieldGroup" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="scaleFieldLabel">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #1A1A1A;
font-family: Open Sans;
font-weight: 600;
font-size: 16px;
line-height: 24px;
letter-spacing: 0%;
</string>
              </property>
              <property name="text">
               <string>Scale</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="scaleComboBox"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="bottomGroup" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>42</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>42</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>16</number>
         </property>
         <property name="leftMargin">
          <number>18</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>18</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>42</height>
            </size>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="applyButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>42</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>42</height>
            </size>
           </property>
           <property name="text">
            <string>Apply &amp;&amp; Restartâ€¦</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
